.github/workflows/test-install.yml

```diff
@@ -163,6 +163,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ['3.9', '3.10', '3.11', '3.12']
+        database-backend: [psql, sqlite]
```

Comment:
I would run these tests only with psql, otherwise this matrix will get very large (since we're testing all supported python versions)